modules/geoipbackend/geoipbackend.cc

```diff
@@ -91,6 +93,25 @@ static vector<std::unique_ptr<GeoIPInterface> > s_geoip_files;
 string getGeoForLua(const std::string& ip, int qaint);
 static string queryGeoIP(const Netmask& addr, GeoIPInterface::GeoIPQueryAttribute attribute, GeoIPNetmask& gl);
 
+// validateMappingLookupFormats validates any custom format provided by the
+// user does not use the custom mapping placeholder again, else it would do an
+// infinite recursion.
+bool validateMappingLookupFormats(const vector<string>& formats) {
+  string::size_type cur,last;
+  for (const auto& lformat : formats) {
```

Comment:
The [check-spelling](https://github.com/marketplace/actions/check-spelling) action doesn't like `lformat`, you could use `lookupFormat` or just `format`:
https://github.com/jsoref/pdns/commit/2170772e6e21da5af191065cc5e5f4dbe228c357#commitcomment-39229499

Reply:
done